### PR TITLE
pc98.xml: Metadata cleanups

### PR DESCRIPTION
**Pull request by ArcadeShadow**
Lowercase on descriptive words (like: "Alt", Disk", "Format", "Fix", etc...)

### CODE REVIEW

**Review by cuavas — CHANGES_REQUESTED** [review]
In cases where “Disk 1” etc. are verbatim quotes of the disk labels (seen in the part IDs), please preserve capitalisation in the quoted text (e.g. “Turbo C++ (alt Disk 1)”).

**Comment by ArcadeShadow** [comment]
> In cases where “Disk 1” etc. are verbatim quotes of the disk labels (seen in the part IDs), please preserve capitalisation in the quoted text (e.g. “Turbo C++ (alt Disk 1)”).

Correction made.

File: hash/pc98.xml [hunk 1]
```diff
@@ -3505,7 +3505,7 @@ TODO:
 
 <!-- is this hacked as well? if so it shall be removed -->
 	<software name="advlanda" cloneof="advland">
-		<description>Adventureland (Alt)</description>
+		<description>Adventureland (alt)</description>
```
Comment (angelosa) [inline hash/pc98.xml:3508]: This is technically a "2HD unofficial" version. 
The comment above stating "shall be removed" pisses me off, we don't have enough control over Jp 80s computer scene to dictate what is or isn't "legit" for inclusion in the SW list, as long as it eases cataloguing and regtesting is good for me imho.

Comment (ArcadeShadow) [reply hash/pc98.xml:3508]: > 

Removed the comment

File: hash/pc98.xml [hunk 2]
```diff
@@ -4671,7 +4670,7 @@ TODO:
 	<!-- The main executable (AMARAN2.EXE) is slightly different in this set -->
 	<software name="amarant2a" cloneof="amarant2">
 		<!-- Origin: Neo Kobe Collection -->
-		<description>Amaranth II (Alt System Disk)</description>
+		<description>Amaranth II (alt system disk)</description>
```
Comment (cuavas) [inline hash/pc98.xml:4673]: “System Disk” is a quote from a disk label.

File: hash/pc98.xml (hunk 1 above)
Comment (cuavas) [reply hash/pc98.xml:3508]: It should still have some kind of comment indicating it’s an unofficial conversion to high density.

File: hash/pc98.xml [hunk 3]
```diff
@@ -14622,7 +14621,7 @@ TODO:
 	</software>
 
 	<software name="dinosaura" cloneof="dinosaur">
-		<description>Dinosaur (Alt Program Disk)</description>
+		<description>Dinosaur (alt Program disk)</description>
```
Comment (cuavas) [inline hash/pc98.xml:14624]: “Disk” is part of the quoted text in this case.

File: hash/pc98.xml [hunk 4]
```diff
@@ -32687,7 +32686,7 @@ Requires MS-DOS 5.00H plus an unknown procedure (HDD install?)
 	</software>
 
 	<software name="3dgolfa" cloneof="3dgolf">
-		<description>New 3D Golf Simulation - Harukanaru Augusta (Alt System Disk)</description>
+		<description>New 3D Golf Simulation - Harukanaru Augusta (alt system disk)</description>
```
Comment (cuavas) [inline hash/pc98.xml:32689]: “System Disk” is a quoted disk label.

File: hash/pc98.xml (hunk 1 above)
Comment (ArcadeShadow) [reply hash/pc98.xml:3508]: Added `notes` with this information

**Comment by ArcadeShadow** [comment]
Thanks for the warnings. Corrected all the disk name with capitalized initial.

**Pull request merged**